src/sqlfluff/dialects/dialect_mysql.py

```diff
@@ -1809,3 +1812,23 @@ class OptimizeTableStatementSegment(BaseSegment):
             Ref("TableReferenceSegment"),
         ),
     )
+
+
+class UpdateStatementSegment(BaseSegment):
+    """An `Update` statement.
+
+    As per https://dev.mysql.com/doc/refman/8.0/en/update.html
+    """
+
+    type = "update_statement"
+    match_grammar: Matchable = StartsWith("UPDATE")
+    parse_grammar: Optional[Matchable] = Sequence(
```

Comment:
There's no benefit to the `StartsWith` when it's the first part of `parse_grammar`, and there's no terminatro clause.

```suggestion
    match_grammar: Matchable = Sequence(
```

I agree ANSI, Exasol, and Teradata all have this, but TSQL doesn't. Fancy making them consistent?

Reply:
Sorted 👍